src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext, useRef, useEffect } from 'react';
+import { DispatchContext, StateContext } from '../Store';
+import classNames from 'classnames';
+
+export const TodoHeader: React.FC = () => {
+  const { title, todos } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const focusInput = () => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  };
+
+  const handlerSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title.trim().length) {
```

Comment:
```suggestion
    if (title.trim()) {
```
NI
redundant to check on length. any not empty string is truthy value